tests/parser/SummaryConfigTests.cpp

```diff
@@ -1950,12 +1950,21 @@ RUNSUM
 
 
 BOOST_AUTO_TEST_CASE(FIPREG) {
-    std::string deck_string = R"(
+    const std::string deck_string = R"(
 -- The FIPREG region has three distinct values, i.e.
 -- there will be three different RPR__REG keywords.
```

Comment:
With the added summary keywords this now holds for other too, right. Maybe update the comment accordingly?

Reply:
> With the added summary keywords this now holds for other too, right. Maybe update the comment accordingly?

Good point.  I've pushed an update to this effect.